steely/plugins/bible/main.py

```diff
@@ -0,0 +1,45 @@
+import json
+import random
+
+from plugin import create_plugin, PluginManager
```

Comment:
`PluginManager` doesn't need to be imported.

(I realise it's also imported in `/nw`, I'll remove it at some stage)

Reply:
My copy and pasting has been revealed.